.github/actions/patch-nightly-version/action.yaml

```
@@ -0,0 +1,11 @@
+# Copyright 2022 Mattermost, Inc.
+name: "patch-version"
+description: This action is used to patch package.json version with the nightly build
+
+runs:
+  using: "composite"
+  steps:
+    - name: ci/generate-version
+      id: generate-version
+      shell: bash
+      run: go run ${{ github.action_path }}/patch-version.go .
```

Could we just run something like the following, and remove all this GoLang clutter? 

```
cat <<< $(jq '.version = (.version | split("-") | first | sub("$"; "-nightly.") | sub("$"; (now | strflocaltime("%Y%m%d"))))' package.json) > package.json
```

```suggestion
      run: cat <<< $(jq '.version = (.version | split("-") | first | sub("$"; "-nightly.") | sub("$"; (now | strflocaltime("%Y%m%d"))))' package.json) > package.json
```